packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -252,7 +253,9 @@ export class SummaryGenerator {
 			// If failure happened on upload, we may not yet realized that socket disconnected, so check
 			// offlineError too.
 			const category =
-				cancellationToken.cancelled || error?.errorType === DriverErrorType.offlineError
+				cancellationToken.cancelled ||
```

Comment:
As per my comment [here](https://github.com/microsoft/FluidFramework/pull/15036#issuecomment-1507303691), we should not do this. The second summary attempt will always fail so don't even do it.